[WIP] Add breadcrumb support

Detect opened the pull request:
Wondering if this is the backbone way to do breadcrumbs? See updated views and `App.setBreadcrumb`

Detect commented:
@rjsteinert do you mind taking a look when convenient? thanks

rjcorwin commented:
That looks pretty good. Only thing I might change is renaming the breadcrumb views to something like `BeeBreadcrumb` and `BeesBreadcrumb` unless you have bigger plans for those views.

Detect committed:
Add breadcrumb support

Review fix

Update breadcrumbs for Bees, Bee, BeeForm, Sensor, SensorForm

Finish breadcrumbs, remove App.setTitle
